packages/perseus/src/types.ts

```diff
@@ -94,6 +95,8 @@ export type ChangeHandler = (
         selected?: number;
         // perseus-all-package/widgets/grapher.jsx
         plot?: any;
+        // Interactive Graph callback (see legacy: interactive-graph.tsx)
+        graph?: InteractiveGraphState;
```

Comment:
Eventually, it would be really  nice if each widget that had an `onChange` callback would type it using a generic. For now, this is fine.